include/ccf/pal/attestation.h

```diff
@@ -119,76 +123,21 @@ namespace ccf::pal
           "Cannot verify SEV-SNP quote if node is virtual");
       }
 
-      auto quote =
-        *reinterpret_cast<const snp::Attestation*>(quote_info.quote.data());
-
-      std::copy(
-        std::begin(quote.report_data),
-        std::end(quote.report_data),
-        report_data.begin());
-      std::copy(
-        std::begin(quote.measurement),
-        std::end(quote.measurement),
-        unique_id.begin());
-
-      auto certificates = crypto::split_x509_cert_bundle(std::string(
-        quote_info.endorsements.begin(), quote_info.endorsements.end()));
-      auto chip_certificate = certificates[0];
-      auto sev_version_certificate = certificates[1];
-      auto root_certificate = certificates[2];
-
-      auto root_cert_verifier = crypto::make_verifier(root_certificate);
-
-      if (
-        root_cert_verifier->public_key_pem().str() !=
-        snp::amd_milan_root_signing_public_key)
-      {
-        throw std::logic_error(fmt::format(
-          "The root of trust public key for this attestation was not "
-          "the expected one {}",
-          root_cert_verifier->public_key_pem().str()));
-      }
-
-      if (!root_cert_verifier->verify_certificate({&root_certificate}))
+      try
       {
-        throw std::logic_error(
-          "The root of trust public key for this attestation was not "
-          "self signed as expected");
+        using namespace ravl;
```

Comment:
Could we avoid this here to make sure it's clear which calls are to `ravl`?

Reply:
Sure, np!